.gitlab-ci.yml

```diff
@@ -18,14 +33,14 @@ variables:
 # us-west1-a
 
 before_script:
-    - pip install -r tests/requirements.txt
+    - /usr/bin/python -m pip install -r tests/requirements.txt
```

Comment:
why is this required?

Reply:
with the new docker image, the default python sued is in /usr/local/bin/python
but ansible has hardcoded /usr/bin/python, then it doesn't find the installed  dependencies.
https://github.com/ansible/ansible/issues/15019#issuecomment-200473572